.github/workflows/verify-pr-labels.yml

```diff
@@ -0,0 +1,25 @@
+name: Verify Pull Request Labels
+
+on:
+  pull_request:
+    types: [ready_for_review, review_requested, labeled, unlabeled]
+
+jobs:
+  verify_pr_labels:
+    if: github.event.pull_request.draft == false
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install github actions dependencies
+        uses: bahmutov/npm-install@v1.6.2
```

Comment:
Do we need to install npm to run this action? Does it work if we remove this? (I think the ubuntu image should have it by default but I might be wrong)

Reply:
Test result:
Removing 'uses: bahmutov/npm-install@v1.6.2', the workflow does not run.
Removing the whole 'Install github actions dependencies' step, the workflow errors with a dependency not found.